acme/client.go

```diff
@@ -432,13 +434,9 @@ func (c *Client) chooseSolvers(auth authorization, domain string) map[int]solver
 				logf("[INFO][%s] acme: Could not find solver for: %s", domain, auth.Challenges[idx].Type)
 			}
 		}
-
-		// If we can solve the whole combination, return the solvers
-		if len(solvers) == len(combination) {
-			return solvers
-		}
 	}
-	return nil
+
+	return solvers
```

Comment:
The current behavior is to try one solver (more or less at random, depending on the map order for each execution). The change doesn't actually work, but the intention was to make it try both solvers and succeed as soon as one works.